lib/collection/src/collection_manager/optimizers/merge_optimizer.rs

```diff
@@ -139,7 +139,7 @@ impl SegmentOptimizer for MergeOptimizer {
         if candidates.len() < 3 {
             return vec![];
         }
-        log::debug!("Merge candidates: {:?}", candidates);
+        log::debug!("Possible merge candidates: {:?}", candidates);
```

Comment:
Let's do trace because we still have a lot of logging now:

```suggestion
        log::trace!("Possible merge candidates: {:?}", candidates);
```